infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -286,14 +286,16 @@ private void prefixTrim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext
     @ServerHandler(type = CorfuMsgType.READ_REQUEST)
     private void read(CorfuPayloadMsg<ReadRequest> msg, ChannelHandlerContext ctx, IServerRouter r) {
         long address = msg.getPayload().getAddress();
-        log.trace("read: {}", msg.getPayload().getAddress());
+        boolean cacheable = msg.getPayload().isCacheOnServer();
```

Comment:
isCacheOnServer, suggests if there is a cache on the server. Instead I believe that you want to refer to the fact that if the read should be cachedOnServer? am I right? if so can we properly rename.

Reply:
Well, the Lombok generated Getter() will add a "is" prefix, which makes the name "cacheOnServer" confusing. I have renamed it to "cacheableOnSever", which makes more sense.